pulsar/table_view_impl.go

```diff
@@ -242,17 +242,20 @@ func (tv *TableViewImpl) Close() {
 }
 
 func (tv *TableViewImpl) handleMessage(msg Message) {
-	tv.dataMu.Lock()
-	defer tv.dataMu.Unlock()
+	if msg.Key() != "" {
```

Comment:
Minor suggestion to just return if the key is empty instead of nesting in the if.
```suggestion
	if msg.Key() == "" {
		return
	}
```